packages/scripts/config/webpack.config.js

```diff
@@ -49,7 +48,10 @@ const cssLoaders = [
 			...( ! hasPostCSSConfig() && {
 				postcssOptions: {
 					ident: 'postcss',
-					plugins: postcssPlugins,
+					sourceMap: ! isProduction,
+					plugins: isProduction
+						? [ ...postcssPlugins, require( 'cssnano' )() ]
```

Comment:
@ocean90, should we provide some default configuration for cssnano?
We could add a similar check as for PostCSS config, there are some ways to include the config for cssnano according to https://cssnano.co/docs/config-file/.

Reply:
I think we can use the same preset as used in https://github.com/WordPress/gutenberg/blob/44212878dcbf6f4616266eb0fb4756293a471615/tools/webpack/shared.js#L84-L91
([Related docs](https://github.com/cssnano/cssnano/tree/master/packages/postcss-discard-comments#removeall) for `removeAll`)

If we go with this adding a check like `hasPostCSSConfig()` makes sense.